src/PersonaWithCard.tsx

```diff
@@ -67,28 +68,31 @@ export class PersonaWithCard extends React.Component<IPersonaWithCardProps, IPer
 
     public render(): JSX.Element {
         return (
-            <HoverCard
-                expandingCardProps={{
-                    onRenderCompactCard: this.renderCompactCard,
-                    onRenderExpandedCard: this.renderExpandedCard,
-                    compactCardHeight: this.state.cardShowMode === CardShowModes.Summary ? 145 : 145 - 60,
-                    expandedCardHeight: this.state.cardShowMode === CardShowModes.Summary ? 385 : 385 + 60
-                }}
-                onCardVisible={() => this.onCardVisible()}
-                instantOpenOnClick={true}
-                trapFocus={true}
-                openHotKey={KeyCodes.enter}
-                className="persona-hover-card"
-            >
-                <Persona
-                    id={this.props.id}
-                    email={this.props.email}
-                    displayName={this.props.displayName}
-                    showMode={this.props.showMode}
-                    size={this.props.size}
-                    className="contact-card-persona"
-                />
-            </HoverCard>
+            <div ref={this.targetElementRef} data-is-focusable={true} tabIndex={0}>
```

Comment:
Minor:
If you have a constant it's better not to use the embedded expressions (easier to read and slightly faster performance). In this case you can try this instead:
```<div ref={this.targetElementRef} data-is-focusable="true" tabIndex="0">```

Reply:
Thanks! changed data-is-focusable.
tabIndex needs to be a number, getting a TS Error: Type 'string' is not assignable to type 'number | undefined'.ts(2322)